Windows tests failing: with permission errors due to file access concurrency problems

Bunch of tests failing:

```
=========================== short test summary info ===========================
FAILED test/test_cli.py::TestCLI::test_update_flags - aiohttp.client_exceptio...
FAILED test/test_cli.py::TestCLI::test_update - PermissionError: [WinError 32...
FAILED test/test_cli.py::TestCLI::test_unknown_warning - PermissionError: [Wi...
FAILED test/test_cli.py::TestCLI::test_quiet_mode - PermissionError: [WinErro...
FAILED test/test_cli.py::TestCLI::test_config_file[D:\\a\\cve-bin-tool\\cve-bin-tool\\test\\config\\cve_bin_tool_config.toml]
FAILED test/test_cli.py::TestCLI::test_config_file[D:\\a\\cve-bin-tool\\cve-bin-tool\\test\\config\\cve_bin_tool_config.yaml]
FAILED test/test_cli.py::TestCLI::test_severity - PermissionError: [WinError ...
FAILED test/test_cli.py::TestCLI::test_CVSS_score - PermissionError: [WinErro...
FAILED test/test_cli.py::TestCLI::test_SBOM - PermissionError: [WinError 32] ...
FAILED test/test_cli.py::TestCLI::test_console_output_depending_reportlab_existence
ERROR test/test_cvedb.py::TestCVEDB::test_refresh_nvd_json - PermissionError:...
======= 10 failed, 11 passed, 10 warnings, 1 error in 97.18s (0:01:37) ========
```

A lot of the error messages look like this:
```
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\runneradmin\\.cache\\cve-bin-tool\\gad\\gemnasium-db-master-conan\\conan\\bison\\CVE-2020-14150.yml'
```

I'm wondering if we've got some tests that should not be run in parallel happening in the wrong job?  These look like gad errors, so I'm going to ping @rhythmrx9 in case he's got some insights.

--- a/cve_bin_tool/cvedb.py
+++ b/cve_bin_tool/cvedb.py
@@ -18,7 +18,7 @@ import requests
 from rich.progress import track
 
 from cve_bin_tool.async_utils import run_coroutine
-from cve_bin_tool.data_sources import curl_source, nvd_source, osv_source
+from cve_bin_tool.data_sources import curl_source, gad_source, nvd_source, osv_source
 from cve_bin_tool.error_handler import ErrorMode
 from cve_bin_tool.log import LOGGER
 from cve_bin_tool.version import check_latest_version
@@ -41,7 +41,12 @@ class CVEDB:
     CACHEDIR = DISK_LOCATION_DEFAULT
     BACKUPCACHEDIR = DISK_LOCATION_BACKUP
     LOGGER = LOGGER.getChild("CVEDB")
-    SOURCES = [nvd_source.NVD_Source, curl_source.Curl_Source, osv_source.OSV_Source]
+    SOURCES = [
+        nvd_source.NVD_Source,
+        curl_source.Curl_Source,
+        osv_source.OSV_Source,
+        gad_source.GAD_Source,
+    ]
 
     def __init__(
         self,

--- a/test/test_cvedb.py
+++ b/test/test_cvedb.py
@@ -16,8 +16,9 @@ class TestCVEDB:
     @classmethod
     def setup_class(cls):
         cls.nvd = nvd_source.NVD_Source(nvd_type="json")
-        cls.cvedb = cvedb.CVEDB(sources=[cls.nvd])
-        cls.nvd.cachedir = tempfile.mkdtemp(prefix="cvedb-")
+        cachedir = tempfile.mkdtemp(prefix="cvedb-")
+        cls.cvedb = cvedb.CVEDB(sources=[cls.nvd], cachedir=cachedir)
+        cls.nvd.cachedir = cachedir
 
     @classmethod
     def teardown_class(cls):

